src/grid/grid.css

```diff
@@ -2,35 +2,36 @@
 
 @value breakpoint-small, breakpoint-middle, breakpoint-large  from "../global/global.css";
 @value large-screen-media, middle-screen-media, small-screen-media from "../global/global.css";
-@value gutterWidth: (var(--ring-unit) * 2);
-@value gutterCompensation: calc(gutterWidth / -2);
-@value outerMargin: calc(var(--ring-unit) * 2);
-@value containerSmall: calc(breakpoint-small + gutterWidth);
-@value containerMedium: calc(breakpoint-middle + gutterWidth);
-@value containerLarge: calc(breakpoint-large + gutterWidth);
-@value width-1: 8.3333%;
-@value width-2: 16.6667%;
-@value width-3: 25%;
-@value width-4: 33.3333%;
-@value width-5: 41.6667%;
-@value width-6: 50%;
-@value width-7: 58.3333%;
-@value width-8: 66.6667%;
-@value width-9: 75%;
-@value width-10: 83.3333%;
-@value width-11: 91.6667%;
-@value width-12: 100%;
 
 .container-fluid,
 .container {
+  --ring-grid-gutter-width: calc(var(--ring-unit) * 2);
+  --ring-grid-gutter-compensation: calc(var(--ring-grid-gutter-width) / -2);
+  --ring-grid-outer-margin: calc(var(--ring-unit) * 2);
+  --ring-grid-container-small: calc(breakpoint-small + var(--ring-grid-gutter-width));
+  --ring-grid-container-medium: calc(breakpoint-middle + var(--ring-grid-gutter-width));
+  --ring-grid-container-large: calc(breakpoint-large + var(--ring-grid-gutter-width));
+  --ring-grid-width-1: 8.3333%;
+  --ring-grid-width-2: 16.6667%;
+  --ring-grid-width-3: 25%;
+  --ring-grid-width-4: 33.3333%;
+  --ring-grid-width-5: 41.6667%;
+  --ring-grid-width-6: 50%;
+  --ring-grid-width-7: 58.3333%;
+  --ring-grid-width-8: 66.6667%;
+  --ring-grid-width-9: 75%;
+  --ring-grid-width-10: 83.3333%;
+  --ring-grid-width-11: 91.6667%;
+  --ring-grid-width-12: 100%;
+
```

Comment:
@andrey-skl restricting those to the .container scope is a breaking change if you use Row outside of Grid. It should be at least mentioned in the changelog but a safer option would be to define them all on `:root` level, for all the components

Reply:
Thanks! Didn't think that `<Row>` may be useful outside of grid

Reply:
I'm not sure we actually use it that way, maybe it's just storybook examples. So maybe `:root` would be an overkill because it would bloat the devtools

Reply:
I would also vote for keeping them on `.container` level to avoid polluting global variables namespace

Reply:
> Maybe something like this could work better:

I like that, let's do so, thanks for the idea